qiskit/circuit/gate.py

```diff
@@ -100,15 +100,15 @@ def control(
         """Return controlled version of gate. See :class:`.ControlledGate` for usage.
 
         Args:
-            num_ctrl_qubits: number of controls to add to gate (default=1)
+            num_ctrl_qubits: number of controls to add to gate (default:: 1)
             label: optional gate label
             ctrl_state: The control state in decimal or as a bitstring
-                (e.g. '111'). If None, use 2**num_ctrl_qubits-1.
+                (e.g. ``'111'``). If None, use 2**``num_ctrl_qubits``-1.
```

Comment:
It might be nicer to also wrap the `**` in code, since it's also Python syntax
```suggestion
                (e.g. ``'111'``). If None, use ``2**num_ctrl_qubits - 1``.
```

Reply:
You are right. Things in this entry need another pass. Checked in a504bb0c3fb9ccd722ac172906e9932f6098a98d